src/main/markdown/doc/latest/DevGuideCodingBasicsJSNI.md

```diff
@@ -1,6 +1,8 @@
 JSNI
 ===
 
+(INFO: For new implementations use [JsInterop](DevGuideCodingBasicsJsInterop.html) instead, to pofit from the benfits like type-safety in your Java Classes.) 
```

Comment:
Aside from the typos, I'd rather frame this as being future-proof: JSNI won't be there in GWT 3 (for things that you cannot do with JsInterop –there are–, then there will be a solution, but it won't be JSNI)